### PR TITLE
Update dialect/eg.js to include last 3 commits

### PR DESCRIPTION
**Pull request by d-zan**
## Description

Update `unban`, `giveaway`, `disconnect`

## Motivation and Context

**Including the last 3 commits from main
Update text**

## Summary of Changes

- Update `unban` to include `mass-unban` 
- Update `giveaway.start.participants`
- Update `disconnect.no_premium`

## Type of Change

- [ ] Bug fix
- [ ] New feature
- [x] Documentation update
- [ ] Refactoring
- [ ] Infrastructure/CI

## Formatting and Review Constraints

- [ ] Code processed via mandatory project formatting tools.
- [ ] Logic architecture validated against domain standards.
- [ ] Specialized domain expert review requested.

## Execution Checklist

- [x] Changes are atomic and strictly scoped.
- [ ] Tests validate specific edge cases.
- [ ] Existing test suite executes without regression.
- [ ] Local documentation reflects structural changes.

## Related Issues

Nothing

### CODE REVIEW

File: dialect/eg.js [hunk 1]
```diff
@@ -57,13 +57,13 @@ module.exports = {
...
         unbanned: `**${correct} تم فك البان من المستخدم {{user}} بنجاح. \n${reason} السبب: {{reason}}**`,
         error: `**${incorrect} فشل في إلغاء حظر المستخدم. الخطأ: {{error}}**`,
-        no_premium: ``,
-        noBans: ``,
-        massTitle: ``,
-        massDesc: ``,
-        massSuccess: ``,
-        massFailed: ``,
-        massReason: ``,
+        no_premium: `**${warning}لازم يكون متوفر اشتراك بريمير، حدث خطتك واستخدم الامر مره اخري.**`,
```
Comment (ameencfw) [inline dialect/eg.js]: اشتراك بريميم وليس Adobe Premiere

File: dialect/eg.js [hunk 2]
```diff
@@ -408,7 +408,7 @@ module.exports = {
         mass_title: `طرد الجميع`,
         mass_desc: `تم طرد جميع الاعضاء من **{channel}**.`,
         success: `تم طرد من الفويس.`,
-        no_premium: ``
+        no_premium: `العضو مش مشترك بريمير.`
```
Comment (ameencfw) [inline dialect/eg.js]: والله بريميم ومش Adobe Premiere

File: dialect/eg.js (hunk 1 above)
Comment (d-zan) [reply dialect/eg.js]: Fixed https://github.com/Velvra/locale/commit/aa32272d66ab2d06755a8ee8775e452730a51620

File: dialect/eg.js (hunk 2 above)
Comment (d-zan) [reply dialect/eg.js]: Fixed 
https://github.com/Velvra/locale/commit/aa32272d66ab2d06755a8ee8775e452730a51620
